src/Raven.Server/Documents/AI/Embeddings/EmbeddingsHelper.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.HashFunction;
+using System.Data.HashFunction.Blake2;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Raven.Client.Documents.Indexes.Vector;
+using Raven.Client.Documents.Queries.Vector;
+using Raven.Server.Documents.ETL.Providers.AI;
+using Raven.Server.Documents.ETL.Providers.AI.Embeddings;
+
+namespace Raven.Server.Documents.AI.Embeddings;
+
+public static class EmbeddingsHelper
+{
+    /// <summary>
+    /// Contains predefined test values used to verify connection with Language Models.
+    /// </summary>
+    /// <remarks>
+    /// These values are sent to the language model when testing the connection
+    /// through either the "Test Connection" button in the UI or the corresponding API endpoint.
+    /// The system uses these values to generate test embeddings and validate that the
+    /// language model properly responds to embedding generation requests.
+    /// </remarks>
+    internal static readonly List<string> ValuesListToVerifyConnection = ["TestValue", "TestValue2"];
+    private static readonly IBlake2B Hash;
+
+    static EmbeddingsHelper()
+    {
+        Hash = Blake2BFactory.Instance.Create(new Blake2BConfig
```

Comment:
Why do we have a factor here, or a configuration for this?
Why are we using an inteface?

`Blake2` is available from `Sodium` directly, why not call that?

Reply:
on me, entirely forgot that the generic hash there is isuing blake2b

Reply:
done